packages/fluentui/react-northstar/src/themes/teams/getPointerStyles.ts

```diff
@@ -173,29 +173,29 @@ export const getPointerStyles = (options: GetPointerStylesOptions): ICSSInJSStyl
 
           left: gap,
           bottom: `calc(${width} - ${height} + ${borderSize})`,
-          transform: 'rotate(90deg) /* @noflip */',
+          transform: `rotate(${rtl ? -90 : 90}deg)`,
```

Comment:
Please revert changes for `top` and `bottom` flipping there is nonsense:

```ts
function styles(rtl) {
  return { transform: `rotate(${rtl ? -90 : 90}deg)` }
}

styles(false) // rotate(90deg)
rtlCssJS(styles(true)) // rotate(90deg)
```

Reply:
Done